addons/mail/static/src/models/emoji_grid_view.js

```diff
@@ -16,12 +16,28 @@ registerModel({
             if (!this.emojiPickerViewOwner.emojiCategoryBarView.activeCategoryView) {
                 return clear();
             }
+            if (this.emojiPickerViewOwner.emojiSearchBar.currentSearch !== "") {
+                const filtered_emojis = this.emojiPickerViewOwner.emojiCategoryBarView.activeCategoryView.emojiCategory.allEmojis.filter(this._filterEmoji);
+                return insertAndReplace(
+                    filtered_emojis.map(emoji => {
+                        return { emoji: replace(emoji) };
+                    })
+                );
+            }
             return insertAndReplace(
                 this.emojiPickerViewOwner.emojiCategoryBarView.activeCategoryView.emojiCategory.allEmojis.map(emoji => {
                     return { emoji: replace(emoji) };
                 })
             );
```

Comment:
Was already the case before this PR: `emojiViews` is theoretically inefficient, even without a search.
Should use `sort` option, so it's more efficient.